build-utils.js

```diff
@@ -0,0 +1,61 @@
+import childProcess from 'child_process';
+import path from 'path';
+import promisify from 'promisify-node';
+
+export let globPromise = promisify('glob');
```

Comment:
Given that this is a style thing, and we have time constraints, I wouldn't block this but I'm opening a separate issue to discuss naming conventions. #2